pysrc/scripts/evaluation.py

```diff
@@ -39,72 +42,167 @@ def create_evaluation_csv(runs: MetanomeRunBatch, output_file: str, config: Glob
     return output_csv
 
 
-# TODO: Actually implement this
-def make_plots(output_file: str, plot_folder: str, config: GlobalConfiguration) -> str:
-    df = pd.read_csv(os.path.join(os.getcwd(), config.output_folder, output_file + '.csv'))   
-    
-    f, (ax1, ax2) = plt.subplots(1, 2, figsize=(10, 15))
-    sns.despine(f)
+def create_TpFpFn_stacked_barplot(axes : matplotlib.axes.Axes, dataframe: pd.DataFrame, groupby_attr: str) -> matplotlib.axes.Axes:
+    df_grouped = dataframe.groupby(groupby_attr)
+    many_plots = False
+    if len(df_grouped) > 10:
+        many_plots = True
```

Comment:
```suggestion
    many_plots = len(df_grouped) > 10
```
Also for the other occurrences of this.